sensor_msgs/PointCloud2 cloud
---
float32[] center
float32 diameter
float32 top
